allure-commandline/src/main/java/ru/yandex/qatools/allure/CommandLine.java

```diff
@@ -0,0 +1,51 @@
+package ru.yandex.qatools.allure;
+
+import io.airlift.command.Cli;
+import org.slf4j.cal10n.LocLogger;
+import ru.yandex.qatools.allure.command.*;
```

Comment:
install checkstyle plugin to sonar and add rule `checkstyle:com.puppycrawl.tools.checkstyle.checks.imports.AvoidStarImportCheck`